charts/alertmanager/values.yaml

```diff
@@ -57,3 +57,5 @@ customRoutes: []
 #   match_re:
 #     tier: platform
 #     namespace: ^(test-ns.*)
+
+enableNonRFC: false
```

Comment:
Can we be more specific with this variable name?

```suggestion
enableNonRFC1918: false
```

Reply:
changed it in my most recent commits.